.mailmap

```diff
@@ -1391,6 +1391,7 @@ Versus Void <versusvoid@gmail.com>
 ViacheslavP <public.viacheslav@gmail.com>
 Victor Brebenar <v.brebenar@gmail.com>
 Victor Immanuel <chrollolucilfer1402@gmail.com> victor immanuel <89346667+victorchrollo14@users.noreply.github.com>
+Victor Momodu <victoramomodu@gmail.com>
```

Comment:
A failure in `test/authors`  is due to  making commits with the GitHub web UI which always recorded the name as GitHub username and the email as something like you should replace this line with 
`Victor Momodu <victoramomodu@gmail.com><87065459+victorian177@users.noreply.github.com>`